sbt_common/storage/src/main/scala/uk/ac/wellcome/storage/dynamo/DynamoConfig.scala

```diff
@@ -1,3 +1,3 @@
 package uk.ac.wellcome.storage.dynamo
 
-case class DynamoConfig(table: String)
+case class DynamoConfig(table: String, index: Option[String])
```

Comment:
Would it be neater to default this new argument to None, and omit it from the constructor unless used? Do any of the changes below actually use the index parameter?

Reply:
I think I prefer it to not have a default value because it would only be there for the purpose of tests. It's never going to use this default value in actual production code. Also, not having a default forced us to modify all of the fixtures to pass the table.index around, which makes sense I think